src/data/services/LmsApiService.js

```diff
@@ -16,24 +14,41 @@ class LmsApiService {
       nav_depth: 3,
       block_types_filter: 'course,chapter,sequential,vertical',
     };
+
     const outlineUrl = `${this.baseUrl}/api/courses/v1/blocks/?${qs.stringify(options)}`;
+    const jwtToken = getAccessToken();
+
     return axios.get(outlineUrl, {
       withCredentials: true,
       headers: {
-        Authorization: `JWT ${this.jwtToken}`,
+        Authorization: `JWT ${jwtToken}`,
       },
     });
   }
 
   static fetchPortalConfiguration(enterpriseSlug) {
     const portalConfigurationUrl = `${this.baseUrl}/enterprise/api/v1/enterprise-customer-branding/${enterpriseSlug}/`;
+    const jwtToken = getAccessToken();
+
     return axios.get(portalConfigurationUrl, {
       withCredentials: true,
       headers: {
-        Authorization: `JWT ${this.jwtToken}`,
+        Authorization: `JWT ${jwtToken}`,
       },
     });
   }
+
+  static authenticate(email, password) {
+    const loginData = {
+      grant_type: 'password',
+      username: email,
+      password,
+      client_id: 'd7cad6d795f7235e8a3c', // client_id configured from django( localhost admin)
```

Comment:
Now that the config PR has landed can you update this to pull from the ./config and add the client_id to the webpack.dev.config and webpack.prod.config files?